praw/reddit.py

```diff
@@ -1,8 +1,21 @@
 """Provide the Reddit class."""
 import configparser
 import os
+import time
 from itertools import islice
-from typing import IO, Any, Dict, Generator, Iterable, Optional, Type, Union
+from logging import getLogger
+from re import search
```

Comment:
Since re is pretty short, I'd prefer this to be imported just as `re` and then we call `re.search`.